### PR TITLE
Fix lag test

### PR DESCRIPTION
**Pull request by arista-hpandya**
### Description of PR

Skip the lag test if portchannel is None.

Summary:

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [x] 202405

#### What is the motivation for this PR?
We wish to skip the test instead of throwing an assertion when portchannel is not available.

#### How did you do it?
We add a check for portchannel

#### How did you verify/test it?
NA

#### Any platform specific information?
NA

#### Supported testbed topology if it's a new test case?
NA

### Documentation

This is not a new feature or test case that requires documentation

### CODE REVIEW

**Comment by kenneth-arista** [comment]
@arlakshm

File: tests/pc/test_po_update.py [hunk 1]
```diff
@@ -99,7 +99,10 @@ def test_po_update(duthosts, enum_rand_one_per_hwsku_frontend_hostname, enum_fro
             portchannel_members = port_channels_data[portchannel]
             break
 
-    pytest_assert(portchannel and portchannel_members, 'Can not get PortChannel interface for test')
+    if not portchannel:
```
Comment (arlakshm) [inline tests/pc/test_po_update.py:102]: if portchannel is not present wont this check in line 90 skip the test?
```
 port_channels_data = asichost.get_portchannels_and_members_in_ns(tbinfo)
    if not port_channels_data:
        pytest.skip(
            "Skip test as there are no port channels on asic {} on dut {}".format(enum_frontend_asic_index, duthost))
```

Comment (kenneth-arista) [reply tests/pc/test_po_update.py:102]: Yes, you're right. This patch predated https://github.com/sonic-net/sonic-mgmt/pull/7574 which merged last year. So, this PR is not needed.

Comment (arlakshm) [reply tests/pc/test_po_update.py:102]: Thanks @kenneth-arista, can you please close this PR

**Comment by arista-hpandya** [comment]
Closing the PR based on the above discussion.

**Pull request closed without merging**